integration-tests/pom.xml

```diff
@@ -11,6 +11,7 @@
   <packaging>pom</packaging>
   <modules>
     <module>example-project</module>
+    <module>test-project</module>
```

Comment:
Example-project and test-project seem ambiguous. I'll change them later to express the real intention of each use case.